src/vade_evan_bbs.rs

```diff
@@ -686,6 +743,9 @@ impl VadePlugin for VadeEvanBbs {
 
     /// Requests a proof for one or more credentials issued under one or more specific schemas and
     /// is sent by a verifier to a prover.
+    ///
+    /// Proof request can be created from scratch or by using a proof offering as an input argument.
```

Comment:
`offering` -> `proposal`?

```suggestion
    /// Proof request can be created from scratch or by using a proof proposal as an input argument.
```